release-notes/topics/aap-211.adoc

```diff
@@ -0,0 +1,16 @@
+[[aap-2.1.1-intro]]
+= Ansible Automation Platform 2.1.1
+
+Red Hat Ansible Automation Platform simplifies the development and operation of automation workloads for managing enterprise application infrastructure lifecycles. It works across multiple IT domains including operations, networking, security, and development, as well as across diverse hybrid environments. Simple to adopt, use, and understand, Red Hat Ansible Automation Platform provides the tools needed to rapidly implement enterprise-wide automation, no matter where you are in your automation journey.
+
+.Enhancements and bug fixes
+
+* Added OCP 4.10 compatibility
+* Added support for HTTP proxy when pushing execution environment images
+* Added changes to the billing operator to be cluster-scoped
```

Comment:
The billing operator is not customer facing. They can't access it (and should not be aware it even exist). This line should be removed.